modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/thunks/pageLanguageUpdate.es.js

```diff
@@ -153,8 +170,8 @@ export default function pageLanguageUpdate({
 			}
 		)
 			.then((response) => response.json())
-			.then(({pages}) => {
-				const visitor = new PagesVisitor(pages);
+			.then((response) => {
+				const visitor = new PagesVisitor(response.pages);
```

Comment:
@marcosapmf, could we keep the destructuring `.then(({pages}) => {` of pages?

Reply:
@interaminense, I made this change because {pages} on line 156 was conflicting with the one on line 150.